files/json2timeseriesdb

```diff
@@ -0,0 +1,362 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
There were some new changes to this file in https://github.com/puppetlabs/puppet-metrics-viewer/pull/28 that should be synced over. What process should we use to ensure these are both in sync?